libs/openFrameworks/3d/ofCamera.cpp

```diff
@@ -60,7 +60,7 @@ void ofCamera::setupPerspective(bool _vFlip, float fov, float nearDist, float fa
 	ofRectangle orientedViewport = getRenderer()->getNativeViewport();
 	float eyeX = orientedViewport.width / 2;
 	float eyeY = orientedViewport.height / 2;
-	float halfFov = PI * fov / 360;
+	float halfFov = PI * fov / 360.0f;
```

Comment:
any reason to not replace PI internally with `glm::pi<float>()` ? 
( but still keep the define for legacy use )

Reply:
Yeah we can do it. I'll substitute PI and TWO_PI / M_TWO_PI by the glm constants.
maybe we have to have some more glm includes

Reply:
done in latest commit, passing tests. should we merge it?